packages/libs/react-ui/src/components/MaskedValue/utils.ts

```diff
@@ -24,10 +24,12 @@ export const maskValue = (
     ...options,
   };
 
-  if (character.length > 1) {
-    throw new Error('Only one character is allowed');
+  if (_character.length > 1) {
+    console.info('Only one character is allowed');
```

Comment:
I think warn would be more suitable here, since it is a warning

Reply:
Done!